packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -0,0 +1,927 @@
+		},
+		{
+			name: 'import',
+			label: __( 'Import content to the site', __i18n_text_domain__ ),
+			context: [ '/posts' ],
+			callback: commandNavigation( '/import/:site' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: downloadIcon,
+		},
+		{
+			name: 'openWooCommerceSettings',
+			label: __( 'Open WooCommerce settings', __i18n_text_domain__ ),
+			callback: commandNavigation( '/wp-admin/admin.php?page=wc-admin' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.ATOMIC,
+			filterP2: true,
+			icon: <WooCommerceWooLogo className="woo-command-palette" />,
+		},
+		{
+			name: 'openWooCommerceSettings',
+			label: __( 'Open WooCommerce settings', __i18n_text_domain__ ),
+			callback: commandNavigation( '/woocommerce-installation/:site' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			siteType: SiteType.SIMPLE,
+			filterP2: true,
+			icon: <WooCommerceWooLogo className="woo-command-palette" />,
+		},
+		{
+			name: 'manageSettingsGeneral',
+			label: __( 'Manage general settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/wp-admin/options-general.php' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+		{
+			name: 'manageSettingsWriting',
+			label: __( 'Manage writing settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/wp-admin/options-general.php' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+		{
+			name: 'manageSettingsReading',
+			label: __( 'Manage reading settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/wp-admin/options-reading.php' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+		{
+			name: 'manageSettingsDiscussion',
+			label: __( 'Manage discussion settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/wp-admin/options-discussion.php' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+		{
+			name: 'manageSettingsNewsletter',
+			label: __( 'Manage newsletter settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/settings/newsletter/:site' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+		{
+			name: 'manageSettingsPodcast',
+			label: __( 'Manage podcast settings', __i18n_text_domain__ ),
+			context: [ '/settings' ],
+			callback: commandNavigation( '/settings/podcasting/:site' ),
+			capability: SiteCapabilities.MANAGE_OPTIONS,
+			icon: settingsIcon,
+		},
+	];
+
+	return commands
+		.filter( ( command ) => {
+			if ( command?.capability && ! capabilities[ command.capability ] ) {
+				return false;
+			}
+
+			if ( command?.siteType && command.siteType !== siteType ) {
+				return false;
+			}
+
+			if ( command?.isCustomDomain && ! isCustomDomain( window.location.host ) ) {
```

Comment:
For Simple sites, `window.localhost.host` will never be a custom domain since we use a `*.wordpress.com` subdomain in WP Admin.

Reply:
Created a follow-up for fixing later on: https://github.com/Automattic/dotcom-forge/issues/5808